app/src/common/hooks/useOnClickOutside.js

```diff
@@ -16,13 +16,23 @@
 
 import { useEffect } from 'react';
 
-export const useOnClickOutside = (ref, handler) => {
+export const useOnClickOutside = (ref, handler, wrapperParentRef, handlerWrapperParentRef) => {
   useEffect(() => {
     if (!handler) {
       return undefined;
     }
 
     const listener = (event) => {
+      if (ref?.current?.contains(event.target)) {
+        return;
+      }
+
+      if (wrapperParentRef?.current.contains(event.target)) {
+        handler(event);
+      } else if (wrapperParentRef) {
+        handlerWrapperParentRef(event);
+      }
+
       if (!ref?.current?.contains(event.target)) {
```

Comment:
handler(event) or handlerWrapperParentRef(event) can affect this check?
Looks like this check always true